metaci/build/tasks.py

```diff
@@ -229,3 +266,17 @@ def delete_scratch_org(org_instance_id):
         return f"Deleted org instance #{org.id}"
     else:
         return f"Failed to delete org instance #{ord.id}"
+
+
+def launch_one_off_build_worker(build, lock_id: str):
+    """Immediately launch a one-off-build with env-appropriate autoscaler"""
+    config = settings.METACI_LONG_RUNNING_BUILD_CONFIG
+    assert isinstance(
+        config, dict
+    ), "Need to define METACI_LONG_RUNNING_BUILD_CONFIG to run one-off-builds"
```

Comment:
METACI_LONG_RUNNING_BUILD_CONFIG is always a dict, the way it's set up in the settings. I think the validation that's relevant would be in HerokuOneOffBuilder to confirm there's an app_name in the config.

Reply:
It's not always a dict because JSON parsing isn't strongly typed.

But I added the other suggestion you proposed.

bad4d34a1

Reply:
Okay, fair point.